src/NServiceBus.AcceptanceTests/Core/Diagnostics/When_a_message_is_faulted.cs

```diff
@@ -1,11 +1,12 @@
-﻿namespace NServiceBus.AcceptanceTests.Core.Diagnostics
+﻿namespace NServiceBus.AcceptanceTests.Diagnostics
 {
     using System.Threading.Tasks;
     using AcceptanceTesting;
     using AcceptanceTesting.Customization;
     using EndpointTemplates;
     using NUnit.Framework;
 
+    [NonParallelizable] // Ensure only activities for the current test are captured
```

Comment:
I think this test doesn't needs this attribute? This seems to be an older test that doesn't test System.Diagnostics related stuff?

Reply:
We added a convention test to check all the tests in the Diagnostic namespace are run sequentially (to avoid concurrency when reading from metric/span recorders)